### PR TITLE
jansson: remove recipe

### PR DESCRIPTION
**Pull request by sashko**
Recipe has been added since softwarecontainer required newer version
of the package meta-oe has provided, which is not the case anymore.

Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

**Comment by martin-ejdestig** [comment]
Maybe rephrase the commit message a bit? I was confused for a minute or two since it says "remove" and then "add". E.g. change to something like:

Recipe was added since softwarecontainer required a newer version
of the package than what meta-oe provided, which is not the case anymore.

**Comment by sashko** [comment]
@martin-ejdestig updated.

**Comment by thenor** [comment]
The commit message confuses me still. The recipe is removed now since it was temporarily added?

**Comment by sashko** [comment]
@thenor yes, the commit message stated:

    Add recipes for SoftwareContainer and Jansson 2.9
    
    Jansson 2.6 is the minimal version required by SoftwareContainer
    but only 2.4 is supported by openembedded.

**Comment by thenor** [comment]
So the commit message should read "The recipe was originally added since ..." ?
